frontend/src/components/VMediaInfo/VMediaTags.vue

```diff
@@ -1,50 +1,223 @@
 <template>
-  <ul v-if="tags.length && additionalSearchViews" class="flex flex-wrap gap-3">
-    <VTag
-      v-for="(tag, index) in tags"
-      :key="index"
-      :href="localizedTagPath(tag)"
-      :title="tag.name"
-    />
-  </ul>
-  <ul v-else class="flex flex-wrap gap-2">
-    <VMediaTag v-for="(tag, index) in tags" :key="index" tag="li">{{
-      tag.name
+  <div v-if="normalizedTags.length && additionalSearchViews" class="-my-1.5px">
+    <ul
+      ref="tagsContainerRef"
+      :aria-label="$t('mediaDetails.tags.title').toString()"
+      class="flex flex-wrap gap-3 overflow-y-hidden p-1.5px"
+      :class="heightClass"
+    >
```

Comment:
```suggestion
  <div v-if="normalizedTags.length && additionalSearchViews">
    <ul
      ref="tagsContainerRef"
      :aria-label="$t('mediaDetails.tags.title').toString()"
      class="flex flex-wrap gap-3 overflow-y-hidden px-1.5px"
      :class="heightClass"
    >
```

I _think_ this is effectively the same appearance? I might be overlooking something though.

Reply:
> Probably due to the maxDiffPixels setting?

Must be,  as it's set to `2px`. 

To clarify, my suggestion was that I didn't see a visual difference between having the negative vertical margin on the wrapper div and vertical padding on the ul vs. having no vertical margin at all. It could be something weird with rendering on my machine, though! I see the focus state the same both ways.